apps/validate/validate.py

```diff
@@ -207,7 +207,8 @@ def create(self, docs, fields=False, **kwargs):
 
     def _get_profile_schema(self, schema, doc):
         doc['validate'].setdefault('extra', {})  # make sure extra is there so it will validate its fields
-        extra_field_types = {'text': 'string', 'embed': 'dict', 'date': 'date', 'urls': 'list'}
+        extra_field_types = {'text': 'string', 'embed': 'dict', 'date': 'date',
+                             'urls': 'list', 'custom': ['datetime', 'string']}
```

Comment:
not sure this would work in general, I think it would be better to introduce new type `any` without type validation